datalad_catalog/catalog.py

```diff
@@ -29,10 +29,36 @@
 class Catalog(Interface):
     # first docstring line is used a short description in the cmdline help
     # the rest is put in the verbose help and manpage
-    """Generate web-browser-based user interface for browsing metadata of a
-    DataLad dataset.
-
-    (Long description of arbitrary volume.)
+    """Generate web-browser-based user interface for browsing metadata of
+    a DataLad dataset.
```

Comment:
This current summary still reflects an older approach where the focus was on datalad datasets.

I suggest rewording: 

Generate a user-friendly web-based data catalog from structured metadata